src/main.py

```diff
@@ -30,6 +31,17 @@
 cache_service = CacheService(settings)
 
 
+@app.exception_handler(FlagsmithUnknownKeyError)
+async def http_exception_handler(request, exc):
+    return JSONResponse(
+        status_code=403,
```

Comment:
Can we change this to 401? That makes more sense here, I think

Reply:
Absolutely, that makes a lot more sense.